sass/main.scss

```diff
@@ -123,7 +123,7 @@ footer {
 }
 
 .logo {
-  background-image: url("/logo-on-white.svg");
+  background-image: url("/logos/vector/green-on-white.svg");
```

Comment:
It should be green-on-transparent. Did you check the site locally? Install [zola](https://www.getzola.org/) and run `zola serve` in the root folder.